cmd/updater/update.sh

```diff
@@ -201,8 +203,8 @@ function get_updater_url() {
     UPDATER_URL="http://algorand-dev-deb-repo.s3-website-us-east-1.amazonaws.com/releases/stable/f9d842778_3.6.2/install_stable_${OS}-${ARCH}_3.6.2.tar.gz"
     UPDATER_FILENAME="install_stable_${OS}-${ARCH}_3.6.2.tar.gz"
 
-    # if on linux, also set variables for signature and checksum validation
-    if [ "$OS" = "linux" ] && [ "$VERIFY_UPDATER_ARCHIVE" = "1" ]; then
+    # also set variables for signature and checksum validation
+    if [ "$VERIFY_UPDATER_ARCHIVE" = "1" ]; then
```

Comment:
just need to make sure that all of the darwin-related binaries can be verified in this way.